qlib/rl/order_execution/policy.py

```diff
@@ -148,7 +148,11 @@ def __init__(
             action_space=action_space,
         )
         if weight_file is not None:
-            load_weight(self, weight_file)
+            loaded_weight = torch.load(weight_file, map_location="cpu")
+            if "vessel" in loaded_weight:
```

Comment:
It seems that loading weight makes some assumptions about the loaded data than simply loading the data directly.
It looks like a reusable feature.

Will it be better to create a standalone function for it?

Reply:
Sounds good, but I didn't find a good way to do this. Any suggestions?

Reply:
@lihuoran 
If I understand it correctly,  it tries to load a policy state dict from a dumped Trainer.
Do you think it is a good idea to create a static utility function for Trainer with a name like `get_policy_state_dict(ckpt_path)`